src/models/Facility.ts

```diff
@@ -0,0 +1,20 @@
+import { Sequelize } from "sequelize";
+const sequelize = require("sequelize");
+
+const tableName = 'facilities'
+
+const fields = {
+  facilityCode: sequelize.STRING,
+  DHIS2OrganizationalUnitCode: sequelize.STRING,
+  openLMISFacilityCode: sequelize.STRING
+}
+
+const options = {
+  freezeTableName: true,
+  tableName,
+  timestamps: false
+}
+
+export const createProductModel = async (
```

Comment:
should this be createFacilityModel?

Reply:
It Should, I was not using it at this point so I realised at a later stage in my current working branch